lerna.json

```diff
@@ -0,0 +1,9 @@
+{
+  "lerna": "2.11.0",
```

Comment:
Add the `private` key ?

Reply:
Done (but I couldn't find a reference to this `private` key in the Lerna documentation)

Reply:
Oups, sorry, I thought it was the root package.json...